code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -164,6 +165,14 @@
 		return 1
 	return ..()
 
+/datum/game_mode/nuclear/proc/are_operatives_onsite()
+	for(var/datum/mind/operative_mind in syndicates)
+		if(ismob(operative_mind.current))
+			var/mob/M = operative_mind.current
+			if(!M.stat && isarea(M.loc) && M.loc in the_station_areas)
```

Comment:
in statements have to be surrounded by () 99% of the time. I think technically this statement is in the 1% since it basically translates to:

if((!M.stat && isarea(M.loc) && M.loc) in the_state_areas)

which is obviously not what you want.